db/migrate/20221017194346_add_use_attached_file_to_task_file.rb

```diff
@@ -0,0 +1,11 @@
+class AddUseAttachedFileToTaskFile < ActiveRecord::Migration[6.1]
+  def change
+    add_column :task_files, :use_attached_file, :boolean, null: true
+
+    TaskFile.all.each do |file|
+      file.update_attribute(:use_attached_file, file.attachment.attached?)
+    end
+
+    change_column :task_files, :use_attached_file, :boolean, null: false
```

Comment:
Columns with `null: false` usually should have a default, I would say. This is probably `false`, as there is no file attached by default?